mod-source-record-manager-server/src/main/resources/log4j2.properties

```diff
@@ -18,23 +18,23 @@ logger.folio_kafka.level = debug
 logger.folio_kafka.additivity = false
 logger.folio_kafka.appenderRef.stdout.ref = STDOUT
 
-rootLogger.level = info
-rootLogger.appenderRefs = info
+rootLogger.level = debug
```

Comment:
I am concerned that is going against recent logging improvements and not cluttering log files. What is your thinking behind this?

Reply:
Changes in logs related to the testing process. I reverted these changes. Thank you.